### PR TITLE
move MM notification to a separate job step so it's in its own container

### PR DESCRIPTION
**Pull request by nickboldt**
### What does this PR do?

move MM notification to a separate job step

Change-Id: I1c30760aa4886d16a18ba64aa10e0ca5cb7b7576
Signed-off-by: nickboldt <nboldt@redhat.com>

### Screenshot/screencast of this PR
N/A

### What issues does this PR fix or reference?
N/A (or see commit message above for issue number)

### How to test this PR?
N/A

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [ ] [Tests are covering the bugfix](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix)
- [x] [The repository devfile is up to date and works, if one exists](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections What issues does this PR fix or reference and How to test this PR completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -83,6 +83,9 @@ jobs:
     - name: Publish packages to npmjs
       if: github.event.inputs.pushToNpmJs == true 
       run: yarn publish:next
+  notify:
+    runs-on: ubuntu-latest
```
Comment (benoitf) [inline .github/workflows/release.yml:87]: use a fixed version like 18.04 or 20.04

**Review by benoitf — APPROVED** [review]

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -83,6 +83,9 @@ jobs:
     - name: Publish packages to npmjs
       if: github.event.inputs.pushToNpmJs == true 
       run: yarn publish:next
+  notify:
```
Comment (azatsarynnyy) [inline .github/workflows/release.yml:86]: Shouldn't the `notify` job depend on the `build` job? Jobs run in parallel by default.
For running sequentially - https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idneeds

Comment (benoitf) [reply .github/workflows/release.yml:86]: yes correct @azatsarynnyy

**Comment by nickboldt** [comment]
if the long running tests ever finish, please feel free to merge this for me :)

**Pull request merged**